src/order/complete_lattice.lean

```diff
@@ -10,8 +10,8 @@ import order.bounds
 set_option old_structure_cmd true
 open set
 
-universes u v w w₂
-variables {α : Type u} {β : Type v} {ι : Sort w} {ι₂ : Sort w₂}
+universes u v v₂ w w₂
+variables {α : Type u} {β : Type v} {β₂ : Type v} {ι : Sort w} {ι₂ : Sort w₂}
```

Comment:
```suggestion
universes u v v₂ w w₂
variables {α : Type u} {β : Type v} {β₂ : Type v₂} {ι : Sort w} {ι₂ : Sort w₂}
```
Otherwise it doesn't look like `v₂` is being used ?

Reply:
Is there anything breaking if you use `Type*`? Note that we can now write `{α β β₂ Type*}` since the `Type*` bug is fixed.

Reply:
Oh nice! I removed all occurrences of universe variables in this file.